api/api_storage.go

```diff
@@ -22,9 +22,9 @@ import (
 
 	"github.com/filecoin-project/lotus/chain/actors/builtin"
 	"github.com/filecoin-project/lotus/chain/types"
-	"github.com/filecoin-project/lotus/extern/sector-storage/fsutil"
-	"github.com/filecoin-project/lotus/extern/sector-storage/storiface"
-	"github.com/filecoin-project/lotus/extern/storage-sealing/sealiface"
+	"github.com/filecoin-project/lotus/storage/pipeline/sealiface"
+	"github.com/filecoin-project/lotus/storage/sealer/fsutil"
+	storiface "github.com/filecoin-project/lotus/storage/sealer/storiface"
```

Comment:
Redundant alias (one of many). I noticed some for `storiface` and some `ffiwrapper`.